types/src/v22/raw_transactions/mod.rs

```diff
@@ -23,9 +25,9 @@ pub use self::error::DecodeScriptError;
 #[derive(Clone, Debug, PartialEq, Deserialize, Serialize)]
 #[serde(deny_unknown_fields)]
 pub struct DecodeScript {
```

Comment:
All the docs changes in `DecodeScript` are not related to adding the `DecodeScriptSegwit` model. They appear to have been changed from what one version outputs to another's. I would just leave them as they were, or if they are wrong you can change them but do it in a separate commit.